frontend/src/components/upload/modals/showCreateUploadModal.tsx

```diff
@@ -238,8 +273,19 @@ const CreateUploadModalBody = ({
               </Text>
             </>
           )}
+
           <Accordion>
-            <Accordion.Item value="description" sx={{ borderBottom: "none" }}>
+            <Accordion.Item
+              value="description"
+              sx={{
```

Comment:
I think using the React if statement would be a cleaner solution instead of using css to optionally display items. It would look like this:
```tsx
            {!options.isReverseShare || (options.shareOptions.descriptionEnabled && (
                <Accordion.Item
                  value="description"
                  sx={{
                    borderBottom: "none",
                  }}
                >
                  <Accordion.Control>Description</Accordion.Control>
                  <Accordion.Panel>
                    <Stack align="stretch">
                      <Textarea
                        variant="filled"
                        placeholder="Note for the recepients"
                        {...form.getInputProps("description")}
                      />
                    </Stack>
                  </Accordion.Panel>
                </Accordion.Item>
              ))}
```

Change my mind if you don't agree :)

Reply:
Yes, we can change this indeed.